project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -62,4 +77,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Event in database
     response.getWriter().println(CommonUtils.convertToJson(SpannerTasks.getEventById(event.getId()).get().toBuilder().build()));
   }
+
+  /* Calls to get NLP Suggested filters if applicable (20 words or more)
+   * @param text: String of text that includes event name and description
+   * @return categoryNames: returns selected names of labels that NLP API suggests for text if available, else empty list
+   */
+  public ArrayList<String> getNlpSuggestedFilters(String text, ArrayList<String> categoryNames) throws IOException{
```

Comment:
Made a new function to make it more test/mock friendly, but let me know if there is a better alternative

Reply:
Why you need to make this function public?

Reply:
Does not need to be public, made private thanks!